article.html

```diff
@@ -10,12 +10,12 @@
 			<p>This will be the nav</p>
 		</header>
 		
-		<div class="wrapper">
+		<div class="article-wrapper">
             <!-- This div is for the template for all articles -->
             <div class="article">
+                <p>Category</p>

```

Comment:
This version would provide text to let the user know which category the article belongs to (e.g. "oily skin" category).